docker-compose.yml

```
@@ -2,7 +2,7 @@ services:
   siren:
     image: sigp/siren
     ports:
-      - '4443:443' # comment this line when using `SSL_ENABLED=false`
+      - '443:443' # comment this line when using `SSL_ENABLED=false`
     #      - "4080:80"  # uncomment this line when using `SSL_ENABLED=false`
```

```suggestion
    #      - "80:80"  # uncomment this line when using `SSL_ENABLED=false`
```
let's go for default ports across the board, then